### ISSUE TITLE
Linux: pid is not set for app in LinuxUtil.get(App app)

### ISSUE THREAD

**Issue by ehsan007** [RaiMan/SikuliX1#144]
PID should be set in the following block: 

```
//check if PID exists, if not throw exception
      executor.execute(command);
      _app.setPID(pid);_
      return app;
```
It's currently not being set resulting in not finding the app on Linux platform.

**Comment by RaiMan** [comment]
Pushed the fix - not tested.

Please test and feedback. Thanks.

**Comment by RaiMan** [comment]
fixed 2019-12-26 in 2.0.2 and 2.1.0
